docs/manuals/core/entities/pipeline-mgt.md

```diff
@@ -65,20 +65,39 @@ pipeline == pipeline_retrieved
 
 Here the two variables `pipeline` and `pipeline_retrieved` are equal.
 
-# Get pipeline by config id
+# Get pipelines by config id
 
 A pipeline can also be retrieved from a scenario by accessing the pipeline's config_id of the scenario.
 
-```python linenums="1"
-import taipy as tp
-import my_config
+!!! Example
 
-scenario = tp.create_scenario(my_config.monthly_scenario_cfg)
+    ```python linenums="1"
+    import taipy as tp
+    import my_config
 
-# Get the pipelines by config id
-sales_pipeline = scenario.sales
-production_pipeline = scenario.production
-```
+    scenario = tp.create_scenario(my_config.monthly_scenario_cfg)
+
+    # Get the pipelines by config id
+    sales_pipeline = scenario.sales
+    production_pipeline = scenario.production
+    ```
+
+Pipelines can also be retrieved by their config_id by calling `taipy.get_entities_by_config_id()^` with their config_id.
+It returns the list of all existing pipelines instantiated from the config_id provided as a parameter.
```

Comment:
"It" -> "This method"
"config_id provided as a parameter" -> "indicated configuration identifier"